src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -1126,16 +1126,35 @@ export class ResultsViewPageStore {
     readonly germlineConsentedSamples = remoteData<SampleIdentifier[]>({
         await:()=>[this.studyIds],
         invoke: async () => {
-            const studies: string[] = this.studyIds.result!;
-            const ids: string[][] = await Promise.all(studies.map(studyId => {
-                return client.getAllSampleIdsInSampleListUsingGET({
-                    sampleListId: this.getGermlineSampleListId(studyId)
-                });
-            }));
-            return _.flatten(ids.map((sampleIds: string[], index: number) => {
-                const studyId = studies[index];
-                return sampleIds.map(sampleId => ({sampleId, studyId}));
-            }));
+            const studiesWithGermlineConsentedSamples = AppConfig.studiesWithGermlineConsentedSamples;
```

Comment:
we gotta write tests for this.  : (  
shouldn't be too hard if you mock client.  you can do this easily by making a method on store that return client.  then use sinon stub to return a function that returns promise with fake data

Reply:
Added a new function `fetchGermlineConsentedSamples` into `StoreUtils`, will add a test for that soon.